sdk/src/TokenRequestState.cs

```diff
@@ -1,24 +1,19 @@
-﻿using System;
+using System;
 using System.Net;
 using Microsoft.IdentityModel.Tokens;
 using Newtonsoft.Json;
 
-namespace Tokenio
-{
+namespace Tokenio {
     [Serializable]
-    public class TokenRequestState
-    {
-        public static TokenRequestState Create(string csrfTokenHash, string state)
-        {
-            return new TokenRequestState
-            {
+    public class TokenRequestState {
+        public static TokenRequestState Create(string csrfTokenHash, string state) {
+            return new TokenRequestState {
                 CsrfTokenHash = csrfTokenHash,
-                InnerState = state
+                    InnerState = state
```

Comment:
Are you using some automatic formatter? Changes are like are unnecessary and make this PR unnecessarily large and messy.